libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -49,6 +49,13 @@ public class MicrosoftAppCredentials : ServiceClientCredentials
         /// </summary>
         private readonly Lazy<AdalAuthenticator> authenticator;
 
+        /// <summary>
+        /// Creates a new instance of the <see cref="MicrosoftAppCredentials"/> class.
+        /// </summary>
+        /// <param name="appId">The Microsoft app ID.</param>
+        /// <param name="password">The Microsoft app password.</param>
+        public MicrosoftAppCredentials(string appId, string password) : this(appId, password, null) { }
```

Comment:
I don't think this is right. 

The other ctor needs to kill the "default" HttpClient and just require one. Otherwise the runtime can't tell them apart.

Reply:
True. Tests still passed, but I;ll remove the 'default' from the other constructor.

Reply:
This constructor will stay like this, and the second constructor will be 

public MicrosoftAppCredentials(string appId, string password, HttpClient customHttpClient)

without default